src/stores/game.js

```diff
@@ -14,6 +14,45 @@ function queenCount(player) {
   return player.faceCards.reduce((queenCount, card) => queenCount + (card.rank === 12 ? 1 : 0), 0);
 }
 
+//this will eventually be moved to tranlations
+const ruleText = [
+  'Scrap all points',
+  'Scrap target Royal or Glasses eight',
+  'Choose 1 card in the Scrap and put it to your hand',
+  'Your opponent discards two cards of their choice from their hand',
+  'Draw two cards from the deck',
+  'Scrap all Royals and Glasses eights',
+  'Play one of the top two cards of the deck and put the other back (both are revealed)',
+  'Your opponent plays with an open hand (their cards are revealed to you)',
+  "Return target card to its controller's hand. They can't play it next turn",
+  'No effect',
+  'Play on top of target point card to steal it',
+  'Your other cards may only be targeted by scuttles',
+  'Reduces the points you need to win. (1K: 14pts, 2K: 10pts, 3K: 5pts, 4K: 0pts)',
+];
+
+class GameCard {
+  constructor(card) {
+    if (!card) return;
+    const str_rank =
+      {
+        1: 'A',
+        11: 'J',
+        12: 'Q',
+        13: 'K',
+      }[card.rank] ?? card.rank;
+    // Stringify Suit
+    const str_suit = ['♣️', '♦️', '♥️', '♠️'][card.suit];
+    this.createdAt = card.createdAt;
+    this.updatedAt = card.updatedAt;
+    this.id = card.id;
+    this.suit = card.suit;
+    this.rank = card.rank;
+    this.name = str_rank + str_suit;
+    this.ruleText = ruleText[card.rank - 1];
```

Comment:
We also need the attachments, which should already exist on the card coming from the server